internal/testutil/runmain.go

```diff
@@ -0,0 +1,96 @@
+// Copyright 2016 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+package testutil
```

Comment:
Already there.

`// Package testutil provides test helpers for the golang-samples repo.`